src/equations/shallow_water_two_layer_2d.jl

```diff
@@ -0,0 +1,911 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
```

Comment:
I won't comment on this right now since there are comments to be resolved for the 1D version.